packages/code-studio/src/settings/ThemeSectionContent.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback } from 'react';
+import {
+  Item,
+  ItemKey,
+  Picker,
+  ThemePicker,
+  useTheme,
+} from '@deephaven/components';
+import { assertNotNull } from '@deephaven/utils';
+import { useDispatch, useSelector } from 'react-redux';
+import { getSettings, updateSettings, type RootState } from '@deephaven/redux';
+
+export function ThemeSectionContent(): JSX.Element {
+  const theme = useTheme();
+  const settings = useSelector<RootState, ReturnType<typeof getSettings>>(
+    getSettings
+  );
+  const dispatch = useDispatch();
```

Comment:
`useAppDispatch`

Reply:
This doesn't seem to work with the thunk action I'm dispatching

Reply:
I believe this is because we aren't using `configureStore` with redux and are using the old, deprecated `createStore`